docs/offline.md

```diff
@@ -0,0 +1 @@
+The requested page wasn't cache already.
```

Comment:
```suggestion
The requested page wasn't cached.
```

this shouldn't be a markdown file, should be emitted like the 404 page is. i also think it's ok to just not show anything for non-cached pages